GitHubDashboard/Controllers/QueryController.cs

```diff
@@ -54,7 +54,13 @@ public QueryController(GitHubClient gitHubClient)
 
         private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string repository, string milestone, string labels)
         {
-            if (string.IsNullOrWhiteSpace(milestone) || milestone == "any" || milestone == "undefined" || milestone == "*")
+            // "undefined" means that milestone was not set on the incoming URL.  Something
```

Comment:
I still don't understand how milestone can be not set on the incoming URL.  The routing places it as a path separator (up on Line 22 above), so there is no way to pass labels without a milestone for example.